src/views/EnrollmentCheckIn/components/CompletedCheckIn/CompletedCheckIn.module.scss

```diff
@@ -0,0 +1,5 @@
+.completed_checkIn {
+  &_congrats_note {
```

Comment:
Since these are using modules now, you don't have to worry about making super unique names for the CSS classes.  I.E, for this one you can just call it congrats_note, without the completed_checkIn { & part...